contracts/feature-tests/rust-testing-framework-tester/src/lib.rs

```diff
@@ -228,6 +229,15 @@ pub trait RustTestingFrameworkTester: dummy_module::DummyModule {
         sc_panic!("Oh no!");
     }
 
+    /// Tests how is generated type name in proxy
+    #[endpoint]
+    fn type_managed_option(
```

Comment:
Let's move this to basic-features
There are some "echo" files with such endpoints